### PR TITLE
ci: add a label to trigger daily CI in PR

### LINKED ISSUE #665: Add a label to trigger daily CI in PR
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-kvrocks/issues) and found no similar issues.

### Motivation

We could add a label like `need-daily-CI`, and when the label is applied to a PR, all jobs of daily CI will join in the checking sequence of this PR.

### Solution

_No response_

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

### PR DESCRIPTION
**Pull request by gogim1**
close #665

### CODE REVIEW

**Comment by ShooterIT** [comment]
cool, thanks

**Review by tisonkun — CHANGES_REQUESTED** [review]
This change doesn't work because pull_request event doesn't include label changes. See also https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#label.

Also I commented on #665 for an alternative - we may not implement the feature in this way.

File: .github/workflows/kvrocks.yaml [hunk 1]
```diff
@@ -115,6 +115,12 @@ jobs:
           cp $HOME/local/bin/redis-cli tests/tcl/redis-cli
           cd tests/tcl && ./runtest
 
+  daily-ci:
+    if: contains(github.event.pull_request.labels.*.name, 'need-daily-CI')
```
Comment (tisonkun) [inline .github/workflows/kvrocks.yaml:119]:
```suggestion
    if: contains(github.event.pull_request.labels.*.name, 'needs-daily-ci')
```

**Review by tisonkun — COMMENTED** [review]
Sorry. I get it wrong that the condition should be event=pull_request + type=labeled.

However, the condition should be handled in `daily-ci.yaml`, where you can change `paths:` to `types: [labeled]`.

**Comment by tisonkun** [comment]
@GoGim1 I'm integrate this patch with a solution to #691.

**Comment by gogim1** [comment]
thanks everyone!

**Pull request closed without merging**